interface.go

```diff
@@ -234,7 +237,7 @@ type TokenGeneratorKey = handshake.TokenProtectorKey
 // as they are allowed by RFC 8999.
 type ConnectionID = protocol.ConnectionID
 
-// ConnectionIDFromBytes interprets b as a Connection ID. It panics if b is
+// ConnectionIDFromBytes interprets b as a ConnectionID. It panics if b is
```

Comment:
Link?

Reply:
I usually don't link the types when they are part of the function or method signature, since it's right there. But that's just my personal preference -- the standard library often does link the type.